builtin/providers/azurerm/import_arm_servicebus_topic_test.go

```diff
@@ -0,0 +1,34 @@
+package azurerm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccAzureRMServiceBusTopic_importBasic(t *testing.T) {
+	resourceName := "azurerm_servicebus_topic.test"
+
+	ri := acctest.RandInt()
+	config := fmt.Sprintf(testAccAzureRMServiceBusTopic_basic, ri, ri, ri)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testCheckAzureRMServiceBusTopicDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: config,
+			},
+
+			resource.TestStep{
+				ResourceName:            resourceName,
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"resource_group_name"},
```

Comment:
We shouldn't have to ignore this anymore - if you look at PR #9073 for details on how to handle this :)